web/src/components/ZLangPicker/ZLangPicker.vue

```diff
@@ -67,8 +73,8 @@ export default {
     },
   },
   methods: {
-    closeNav() {
-      this.$emit('close-nav', 'null');
+    clickItem() {
+      this.$emit('click-item', null);
```

Comment:
Why emit `null` ?
You can skip it:
```suggestion
      this.$emit('click-item');
```